available_images.md

```diff
@@ -142,12 +142,23 @@ SageMaker Framework Graviton Containers (SM support only)
 
 NVIDIA Triton Inference Containers (SM support only)
 ============================
-**Note**:
+**Versions 23.12 and onwards**:
+1. Starting version 23.12, onwards, the sagemaker tritonserver is available in a different set of accounts than the previous ones. The new accounts are now the same as other DLCs listed on this page, making it easier to switch containers going forward. These accounts are listed in section '# Available Deep Learning Containers Images' above.
```

Comment:
This wording is a bit confusing, and the tables below reference the older version account IDs. In order to find a new image, it seems like a customer will have to construct their own URI by swapping out acct IDs from the table at the top of the file.

Reply:
I've restructured a bit to clarify the versions after 23.12, and those before 23.12